gammapy/image/tests/test_core.py

```diff
@@ -294,3 +294,12 @@ def test_image_pad():
 
     image2 = image.pad(pad_to_factor=4, mode='reflect')
     assert image2.data.shape == (16, 12)
+
+
+def test_skycoord_pixel_conversion():
+    image = SkyImage.empty(nxpix=10, nypix=15)
```

Comment:
I think the most important test to have is one that does an assert on an explicit hard-coded result number.

Keeping the round-tripping test is OK if you like.